build.gradle

```diff
@@ -46,7 +46,7 @@ task clean(type: Delete) {
 }
 
 ext {
-    buildToolsVersion = "29.0.3"
+    buildToolsVersion = "30.0.2"
```

Comment:
Woo-hoo!

Reply:
Dang it...it's still failing in lgtm

```
[2020-09-25 21:35:42] [autobuild] * What went wrong:
[2020-09-25 21:35:42] [autobuild] A problem occurred configuring project ':aws-analytics-pinpoint'.
[2020-09-25 21:35:42] [autobuild] > Failed to install the following SDK components:
[2020-09-25 21:35:42] [autobuild]       build-tools;30.0.2 Android SDK Build-Tools 30.0.2
[2020-09-25 21:35:42] [autobuild]       platforms;android-30 Android SDK Platform 30
[2020-09-25 21:35:42] [autobuild]   The SDK directory is not writable (/opt/extra/android/versions/latest)
[2020-09-25 21:35:42] [autobuild] * Try:
````

Reply:
[Manoel is also encountering the `The SDK directory is not writable` error](https://github.community/t/android-projects-are-not-working/131143).